code/modules/clothing/chameleon.dm

```diff
@@ -297,7 +298,7 @@
 //*********************
 
 /obj/item/clothing/glasses/chameleon
-	name = "optical meson scanner"
+	name = "Optical Meson Scanner"
```

Comment:
Are mesons themselves capitalized? If yes, then okay. Though they both should be uncapitalized, as they're not unique names.